tuftool/src/error.rs

```diff
@@ -373,12 +351,6 @@ pub(crate) enum Error {
         backtrace: Backtrace,
     },
 
-    #[snafu(display("Failed writing target data to disk: {}", source))]
-    WriteTarget {
-        source: std::io::Error,
-        backtrace: Backtrace,
-    },
```

Comment:
I am surprised that so many errors end up unused.

Over in the core kit, when I was working on signpost, I noticed that a lot of the new "unused" warnings were false positives. Seems to be an issue with a crate that's built as both a library and a binary - the error variants that aren't used in the library side get flagged even though the binary needs them.

Which is to say, please double check via `git grep` or `rg` that all of these are actually not used.

Reply:
I think a couple of these might have just been copied from `tough` at some point. I wasn't able to find any references in `tuftool` itself.